packages/swirl-components/src/components/swirl-popover-trigger/swirl-popover-trigger.tsx

```diff
@@ -145,7 +159,34 @@ export class SwirlPopoverTrigger {
     return isActive;
   }
 
+  private getPopoverAndTriggerEls() {
```

Comment:
Question: Why combining both elements in a single object? I think that makes it more difficult to reason about. I would just call getPopoverEl() and getTriggerEl() where I need it.